Update Readmes in packages (#43)
kyma-incubator/documentation-component: packages/odata-render-engine/README.md

## Prerequisites

- [`odata-react`](https://github.com/kyma-incubator/documentation-component/tree/master/packages/odata-react): ^0.3.0

### Styles

To use default styles based on [SAP Fundamentals](https://sap.github.io/fundamental/), install the npm package from [this](https://sap.github.io/fundamental/getting-started.html) Getting Started guide and import the styles as follows:

```js
import "@kyma-project/odata-react/lib/styles.css";
import "fiori-fundamentals/dist/fiori-fundamentals.min.css";
```

> To use the default font `72` provided by SAP Fundamentals, import (after installation [SAP Fundamentals](https://sap.github.io/fundamental/) package) the font as follows:

Reviewer: ```suggestion
To use the default font `72` provided by SAP Fundamentals, install the [SAP Fundamentals](https://sap.github.io/fundamental/) package, and import the font as follows:
```
